src/dataflow/load.js

```diff
@@ -43,7 +43,7 @@ export function request(target, url, format) {
         df.ingest(target, data, format);
       },
       function(error) {
-        df.error('Loading failed: ' + url, error);
+        df.error('Loading failed: ' + (typeof url === 'string' ? url : JSON.stringify(url)), error);
```

Comment:
A few thoughts:
1. If the url is a string instance, running it through `JSON.stringify` is fine -- it will simply add quotes to it.
2. If an input `url` object has circular references, `JSON.stringify` may raise an exception. The exception is unhandled here and so `pending.done()` will fail to be invoked.